core/server/api/configuration.js

```diff
@@ -27,7 +28,7 @@ function getBaseConfig() {
         fileStorage:    config.get('fileStorage') !== false,
         useGravatar:    !config.isPrivacyDisabled('useGravatar'),
         publicAPI:      config.get('publicAPI') === true,
-        blogUrl:        config.get('url').replace(/\/$/, ''),
```

Comment:
Will reconsider my comment, as we force ssl when secure is true and `urlSSL` is not present.